yarnlockparser/yarnlockparser_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	log "github.com/sirupsen/logrus"
+	"log"
```

Comment:
Just double checking that it is an intended change.

Reply:
oh, it is surely not, thanks !